mountpoint-s3/src/inode.rs

```diff
@@ -838,7 +842,7 @@ impl SuperblockInner {
             InodeKindData::Directory { children, .. } => children.get(name).cloned(),
         };
         match (remote, inode) {
-            (None, None) => Err(InodeError::FileDoesNotExist),
+            (None, None) => Err(InodeError::FileDoesNotExist(name.into())),
```

Comment:
Should we also provide the parent key?

Reply:
I am not sure because in half of the only parent InodeNo. is present. And I dont think that would give much information.